api-tests/dev_apis/internal_trusted_storage/test_s003/test_s003.c

```diff
@@ -46,21 +47,27 @@ int32_t psa_sst_insufficient_space(security_t caller)
     for (i = 0 ; i < NUM_ITERATIONS; i++)
     {
         val->print(PRINT_TEST, "[Check %d] Overload storage space\n", i + 1 );
-        for (uid = 1; status == PSA_SST_SUCCESS; uid++)
+        for (uid = TEST_BASE_UID_VALUE; status == PSA_SST_SUCCESS; uid++)
```

Comment:
As per the doc, UID range is mentioned from 15-20. But here it can go beyond UID value 20

Reply:
Document will be  updated to reflect this.